packages/components/molecules/f-restaurant-card/src/components/subcomponents/DeliveryTimeMeta/_tests/DeliveryTimeMeta.test.js

```diff
@@ -0,0 +1,78 @@
+import { shallowMount, mount } from '@vue/test-utils';
+import sut from '../DeliveryTimeMeta.vue';
+
+describe('DeliveryTimeMeta', () => {
+    it('should be defined', () => {
+        // arrange
+        const propsData = {
+            eta: '20 - 25 min'
+        };
+
+        // act
+        const wrapper = shallowMount(sut, { propsData });
+
+        // assert
+        expect(wrapper.exists()).toBe(true);
+    });
+    it('should show eta text when data is present', () => {
```

Comment:
we currently have unique data-tesst-ids for eta and another for distance and address. I think we should preserve this. These tests are quite coupled to the Icon Text implementation at the moment. 

I think we should keep unique test IDs and check for the presence of those instead of having the same test ID for all icon text instances (a testing nightmare in the future) and avoid inspecting text output .

Reply:
Happy to refactor, it was done because the text content included visible and accessible text and so assertions were harder. But if we move the accessible text to a aria tag, we can avoid having to do this

Reply:
unresolved because I think this is still a legitimate issue with the test approach - I think we should give eta, delivery and address their own unique test IDs and simply look for those. Rather than giving them all the same ID and checking text output